app/Agri.Models/Configuration/FeedForageType.cs

```diff
@@ -0,0 +1,20 @@
+﻿using Newtonsoft.Json;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+using System.Runtime.Serialization;
+
+namespace Agri.Models.Configuration
+{
+    public class FeedForageType : Versionable
+    {
+        [Key]
+        public int Id { get; set; }
+
+        public string Name { get; set; }
+        public int FeedId { get; set; }
+
+        [JsonIgnore]
+        [IgnoreDataMember]
+        public Feed Feed { get; set; }
```

Comment:
Since a FeedForageType is a Foreign Key in Feed, then it belongs to one or more Feeds, thus you need to remove this and replace with.

        public List<Feed> Feeds { get; set; }